client/services/FeedbackGeneratorService.js

```diff
@@ -54,11 +54,12 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     return {
-      getFeedback: function(prompt, codeEvalResult) {
+      getFeedback: function(
+          prompt, codeEvalResult, rawCodeLineIndexes) {
```

Comment:
I missed this, but you didn't update the spec with rawCodeLineIndexes. Did you verify that this doesn't break any tests?